src/test/java/org/opentripplanner/smoketest/DenverSmokeTest.java

```diff
@@ -30,4 +30,49 @@ public void routeFromSouthToNorth() {
       List.of("WALK", "TRAM", "WALK", "BUS", "WALK")
     );
   }
+
+  @Test
+  public void vehiclePositions() throws JsonProcessingException {
```

Comment:
Is it possible to move this to a utils file so that we can test septa realtime positions as well?

Reply:
That would absolutely possible. Could you send me the URL to the vehicle positions feed?